QueueInteropTransport.php

```diff
@@ -103,11 +104,18 @@ public function receive(callable $handler): void
     public function send(Envelope $message): void
     {
         $psrContext = $this->contextManager->psrContext();
-        $destination = $this->getDestination();
-        $topic = $psrContext->createTopic($destination['topic']);
+
+        $topicItem = $message->get(Topic::class);
```

Comment:
I'd prefer to move this logic to the `getDestination` method instead. Can it take the message as a nullable argument?